aio_pika/channel.py

```diff
@@ -6,6 +6,7 @@
 
 import aiormq
 import aiormq.types
+from pamqp import commands
```

Comment:
In other files you are making `import pamqp.commands`, - I think it worth to do the same here so that it'll be the same everywhere.